src/BPSecLib_Private.h

```diff
@@ -581,6 +581,13 @@ int BSL_BundleCtx_CreateBlock(BSL_BundleRef_t *bundle, uint64_t block_type_code,
  */
 int BSL_BundleCtx_RemoveBlock(BSL_BundleRef_t *bundle, uint64_t block_num);
 
+/** @brief Requests dropping of bundle
+ *
+ * @param[in] bundle    Context bundle
+ * @return 0 on success, negative on failure.
+ */
+int BSL_BundleCtx_DeleteBundle(BSL_BundleRef_t *bundle);
```

Comment:
How does this function interact with the `valid` member of that struct. It would be good to clarify in the docs for each one.